Added content to dryville_story.md and updated missing .md extension for course_guide

Megan-Verf opened the pull request:
Added text to dryville_story.md file: 
- Added sections "Getting Water to Your Homes" and "Dryville's First Water Works" 
- Added hyperlinks to text 

Added missing .md extension to course_guide file to improve readability on Github website 

#[issue number]

abnerbog reviewed (changes requested):
Thanks so much for PR review request @Megan-Verf! Great catch with [4d8c655](https://github.com/CUAHSI-GitFlows-Learners/learning-gitflows-Megan-Verf/pull/1/commits/4d8c655c62b8191c18c9929c027254b0d01d436e) -- totally my fault! 

The other changes look just about perfect! There are two missing hyperlinks in the "Getting Water to Your Homes Section" below -- could you add these in another commit?

![image](https://github.com/user-attachments/assets/c4130db0-8e5d-4351-b26f-aa9b71191fac)

Megan-Verf commented:
Thank you for your response @abnerbog! I've updated the hyperlinks under the "Getting Water to Your Homes Section" and added them in another [commit](https://github.com/CUAHSI-GitFlows-Learners/learning-gitflows-Megan-Verf/pull/1/commits/f3e180649ea61f6df7c1e91692b78f12cb54fece). 

Please let me know if there is anything else!

Megan-Verf commented:
Thank you @abnerbog! Is there something I have to do from my end to merge the PR? I don't see any changes to the [upstream repository](https://github.com/CUAHSI-GitFlows-Learners/learning-gitflows-Megan-Verf) yet. As a result, completing Step 9 doesn't identify any updates.

abnerbog commented:
Apologies -- I just merged the PR :)

Megan-Verf commented:
Thank you! I just finished step 9 where I closed the loop between the main repo and my local copy. Please let me know when the content has been added for step 10!

abnerbog commented:
Thanks for the heads up! The content has been added in c49609d03ed4ef8f751472e325383a27258a338a -- you are all set to move on!

Megan-Verf commented:
Great, thank you! I have now arrived at step 11. Please let me know when to proceed.

abnerbog commented:
Awesome, you are set to move on!
